pages/_formats/genome-coordinates.md

```diff
@@ -44,23 +42,23 @@ Similarly, represent positions in APIs using 0‑based coordinates.
 
 Consider a subsequence `GAGTGC` of a larger sequence of bases (which might be a reference chromosome, for example):
 
-<img src="{{ 'assets' | relative_url }}/img/genome-coordinates-img/1-based.svg" alt="Sequence data with 1-based coordinates" />
+![Sequence data with 1-based coordinates](/assets/img/genome-coordinates-img/1-based.svg)
```

Comment:
Hard-coded paths break if moving to a different environment, e.g. a sub-directory of a test server; therefore the Liquid call. Agree that it may be easier for people not running their own build server to see them in the source.

Reply:
In hts-specs we considered that being able to see the images in PRs and in the GitHub repository (as well as on the web site) outweighed the downside of slightly harder web site rearrangements. But okay.